app/controllers/organizations_controller.rb

```diff
@@ -33,6 +37,9 @@ def index
 
   def edit
     @organization = Organization.find params[:id]
+    if current_user.organization_id != @organization.id
```

Comment:
It's generally a good practice in Rails to use objects instead of ids so something like:
```ruby
if current_user.organization != @organization
```